_dashboards/branding.md

```diff
@@ -72,29 +72,27 @@ logo:
 applicationTitle: "My custom application"
 ```
 
-We recommend linking to images that are hosted on a web server, but if you really want to use locally hosted images, save your images inside `src/core/server/core_app/assets`, and then configure `opensearch_dashboards.yml`. You can access locally stored images through the `ui` folder.
+We recommend linking to images that are hosted on a web server, but if you really want to use locally hosted images, save your images inside `assets`, and then configure `opensearch_dashboards.yml` to use the correct paths. You can access locally stored images through the `ui/assets` folder.
 
 The following example assumes the default port of 5601 that Dashboards uses and demonstrates how to link to locally stored images.
 
 ```yml
 logo:
-  defaultUrl: "https://localhost:5601/ui/my-own-image.svg"
-  darkModeUrl: "https://localhost:5601/ui/my-own-image.svg"
+  defaultUrl: "https://localhost:5601/ui/assets/my-own-image.svg"
+  darkModeUrl: "https://localhost:5601/ui/assets/my-own-image.svg"
```

Comment:
NIT: would we be able to put like my-own-image-dark-mode? By default if user only set defaultUrl it will use the same value for dark mode if dark mode url is not set.

Reply:
Yep, that seems clearer. Added.